chia/_tests/wallet/cat_wallet/test_offer_lifecycle.py

```diff
@@ -118,10 +117,10 @@ def generate_secure_bundle(
     offered_amount: uint64,
     tail_str: Optional[str] = None,
 ) -> SpendBundle:
-    announcement_assertions = [a.to_program() for a in announcements]
+    announcement_assertions: List[Any] = [a.to_program() for a in announcements]
     selected_coin_amount = sum([c.amount for c in selected_coins])
     non_primaries = [] if len(selected_coins) < 2 else selected_coins[1:]
-    inner_solution: List[CastableType] = [
+    inner_solution: List[List[Any]] = [
```

Comment:
mypy isn't smart enough to handle this as a `List[Program]` hint?

Reply:
(the announcement_assertions that is)

Reply:
[`d79ac98` (#16898)](https://github.com/Chia-Network/chia-blockchain/pull/16898/commits/d79ac98c017af255c54e9e2af85e0d979bde3f19) And that pointed out an error in the `inner_solution` hint.